common/scala/src/main/scala/org/apache/openwhisk/core/database/MultipleReadersSingleWriterCache.scala

```diff
@@ -445,25 +444,37 @@ trait MultipleReadersSingleWriterCache[W, Winfo] {
   }
 
   /** This is the backing store. */
-  private lazy val cache: ConcurrentMapBackedCache[Entry] = evictionPolicy match {
-    case AccessTime =>
-      new ConcurrentMapBackedCache(
-        Caffeine
+  private lazy val cache: ConcurrentMapBackedCache[Entry] = new ConcurrentMapBackedCache(
```

Comment:
This can be simplified a bit
```suggestion
    private lazy val cache: ConcurrentMapBackedCache[Entry] = createCache()

  private def createCache() = {
    val b = Caffeine
      .newBuilder()
      .softValues()

    evictionPolicy match {
      case AccessTime => b.expireAfterAccess(5, TimeUnit.MINUTES)
      case _          => b.expireAfterWrite(5, TimeUnit.MINUTES)
    }

    if (fixedCacheSize > 0) b.maximumSize(fixedCacheSize)
    new ConcurrentMapBackedCache(b.build().asMap().asInstanceOf[ConcurrentMap[Any, Future[Entry]]])
  }
```